cli/tox.ini

```diff
@@ -59,7 +59,7 @@ commands =
 basepython = python3
 skip_install = true
 deps =
-    isort
+    isort==4.3.21
```

Comment:
Is it a right change?

Reply:
Yes, we need to pin isort to avoid Travis failures; we'll address this better in the future.